### PR TITLE
Feature: makes sidebar width configurable

### PR DESCRIPTION
**Pull request by awcodes**
This feature adds a config key to Filament Config file to support custom sidebar sizes set by the user.

Option can be set in any CSS unit.

```php
'sidebar' => [
    ...
    'width' => '320px'
],
```

### CODE REVIEW

File: packages/admin/config/filament.php [hunk 1]
```diff
@@ -205,6 +205,7 @@
             'groups' => [
                 'are_collapsible' => true,
             ],
+            'width' => '320px'
```
Comment (danharrin) [inline packages/admin/config/filament.php:208]: IMO the default should be null and set in the view instead, so we have the flexibility to customise it ourselves internally if we wish

**Review by danharrin — CHANGES_REQUESTED** [review]

File: packages/admin/config/filament.php (hunk 1 above)
Comment (awcodes) [reply packages/admin/config/filament.php]: updated with latest commit

**Comment by danharrin** [comment]
Needs to be recompiled, then we're good to go

**Comment by awcodes** [comment]
Should be ok now.

**Comment by danharrin** [comment]
Thanks!

**Pull request merged**